generator/src/main/java/org/corfudb/generator/distributions/Operations.java

```diff
@@ -30,12 +32,18 @@ public Operations(State state) {
                 new SnapshotTxOperation(state),
                 new SleepOperation(state),
                 new RemoveOperation(state));
+//              TODO: Fix nestedTx path to enable it
+//              new NestedTxOperation(state));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)